### ISSUE TITLE
Not compatible with content-defender

### ISSUE THREAD

**Issue by yannkuesthardt** [team23/t23_inline_container#6]
**Problem**
The inline CE selector does not filter out Types that are blocked by ichhabrecht/content-defender and therefor throws the following error when trying to save: "1: The record "abc" couldn't be saved due to disallowed value(s)."

Used Versions:
- TYPO3: 11.5.19
- ichhabrecht/content-defender: 3.2.3
- b13/container: 2.1.1
- team23/t23_inline_container: 0.0.8

**Comment by thegass** [comment]
can confirm this issue.
also the initial colpos of new elements is (at least in my system) set to 0 instead of the colpos f. container elements (in my case 210)

![image](https://user-images.githubusercontent.com/101392/204801446-c3784472-be06-40ae-afaa-fc2ed79e8e7a.png)

**Comment by timdreier** [comment]
@yannkuesthardt thanks for reporting. Will implement this as soon as possible.

Thanks @thegass, I already have this in mind. I'll create a separate issue for that.
